lib/src/sdk_api/view/video_view.dart

```diff
@@ -103,23 +122,26 @@ class VideoView extends StatefulWidget {
   /// @internal
   /// 
   /// A constructor that could be used when the [VideoView] is an element in a collection 
-  /// widget, such as a [GridView] or a [ListView]. The constructor requires providing the [Participant]
-  /// who would like to display a [MediaStream], an optional [Key], and the [MediaStreamSelector]. 
+  /// widget, such as a [GridView] or a [ListView]. The constructor requires providing the
+  /// [Participant] who would like to display a [MediaStream], an optional [Key], and
+  /// the [MediaStreamSelector]. 
   /// 
   /// The [MediaStreamSelector] is a closure or an unnamed function that is run internally by the 
   /// widget to decide which [MediaStream] to display. The algorithm from this closure should 
-  /// return the [MediaStream] that should be displayed based on the [List] of [MediaStream]s provided as 
-  /// an argument to the closure. This closure runs when the [VideoView] is created and every time the 
-  /// [MediaStream]s of the related [Participant] change.
-  const VideoView.withMediaStreamSelector({required this.participant, Key? key, required this.mediaStreamSelector})
+  /// return the [MediaStream] that should be displayed based on the [List] of [MediaStream]s
+  /// provided as an argument to the closure. This closure runs when the [VideoView] is created and
+  /// every time the [MediaStream]s of the related [Participant] change.
+  const VideoView.withMediaStreamSelector({
```

Comment:
Another constructor for VideoView? Too much, but that's not the worst. Three constructor completely independent of each other and set different fields caused that objects of the same class behave differently based on fields that set to null or not ( too many if that check which fields are null or not)